vignettes/finalsize.Rmd

```diff
@@ -0,0 +1,275 @@
+The `symmetric = TRUE` argument to `socialmixr::contact_matrix()` returns a symmetric contact matrix, so that the contacts reported by group $\{i\}$ of individuals from group $\{j\}$ are the same as those reported by group $\{j\}$ of group $\{i\}$.
+
+The demographic data --- the number of individuals in each age group --- is also available through `socialmixr::contact_matrix()`.
+
+```{r}
+# get UK polymod data
+polymod <- socialmixr::polymod
+contact_data <- socialmixr::contact_matrix(
+  polymod,
+  countries = "United Kingdom",
+  age.limits = c(0, 5, 18, 40, 65),
+  symmetric = TRUE
+)
+
+# get the contact matrix and demography data
+contact_matrix <- t(contact_data$matrix)
+demography_vector <- contact_data$demography$population
+demography_data <- contact_data$demography
+
+# scale the contact matrix so the largest eigenvalue is 1.0
+contact_matrix <- contact_matrix / max(eigen(contact_matrix)$values)
+
+# divide each row of the contact matrix by the corresponding demography
+contact_matrix <- contact_matrix / demography_vector
+
+n_demo_grps <- length(demography_vector)
+```
+
+::: {.alert .alert-secondary}
+
+### Heterogeneous social mixing {-}
+
+The social contact data used in this example illustrate an important way in which populations may be heterogeneous: the various age groups can have quite different patterns of social contacts.
+
+The method implemented in _finalsize_ accounts for this heterogeneity. Other forms of population heterogeneity are covered in the ["Heterogeneous susceptibility"](varying_susceptibility.html) vignette.
+:::
+
+## Population susceptibility
+
+In the early stages of an epidemic outbreak, a safe assumption might be that all age groups have a similar, high susceptibility to infection. This means that all individuals fall into a single category: highly or fully susceptible.
+
+High uniform susceptibility can be modelled as a matrix with values of 1.0, with as many rows as there are demographic groups. The matrix has a single column, representing the single susceptibility group to which all individuals belong.
+
+```{r}
+# all individuals are equally and highly susceptible
+n_susc_groups <- 1L
+susc_guess <- 1.0
+```
+
+```{r}
+susc_uniform <- matrix(
+  data = susc_guess,
+  nrow = n_demo_grps,
+  ncol = n_susc_groups
+)
+```
+
+Final size calculations also need to know the proportion of each demographic group $\{i\}$ that falls into the susceptibility group $\{j\}$. This distribution of age groups into susceptibility groups can be represented by the demography-susceptibility distribution matrix.
+
+Since all individuals in each age group have the same susceptibility, there is no variation within age groups. Consequently, all individuals in each age group fall into the highly or fully susceptible group. This can be represented as a single-column matrix, with as many rows as age groups, and as many columns as susceptibility groups.
```

Comment:
```suggestion
Since all individuals in each age group have the same susceptibility, there is no variation within age groups. Consequently, all individuals in each age group are assumed to be fully susceptible. This can be represented as a single-column matrix, with as many rows as age groups, and as many columns as susceptibility groups.
```

Reply:
Fixed offline